docs/getting-started/uninstall/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Uninstall Shopify App CLI
+section: getting-started
+toc: false
+redirect_from: "/uninstall/"
+---
+
+Shopify App CLI can be removed from your system using the same package manager you used to install it.
+
+### Ruby gem
+
+```console
+$ gem uninstall shopify-cli
+```
+
+### Brew (MacOS)
+
+```console
+$ brew uninstall shopify-cli
+```
+
+### apt (Debian, Ubuntu)
+
+```console
+$ apt uninstall shopify-cli
```

Comment:
This should be `apt remove shopify-cli`.

Reply:
And add `sudo` in front of it.